includes/spoolchanges.js

```diff
@@ -62,27 +56,31 @@ module.exports = async function(db, log, bufferSize, tolerance = 600000) {
         // Extract the document ID from the change
         return { id: changeResultItem.id };
       } else {
-        throw new error.BackupError('InvalidChange', `Received invalid change: ${changeResultItem}`);
+        throw new error.BackupError('SpoolChangesError', `Received invalid change: ${JSON.stringify(changeResultItem)}`);
       }
     });
   };
 
+  const mapChangesBatchToBackupBatch = function(changesBatch) {
+    return { command: 't', batch: batch++, docs: mapChangesToIds(changesBatch) };
+  };
+
+  const mapBackupBatchToPendingLogLine = function(backupBatch) {
+    totalBuffer += backupBatch.docs.length;
+    debug('writing', backupBatch.docs.length, 'changes to the backup log file with total of', totalBuffer);
+    return `:t batch${backupBatch.batch} ${JSON.stringify(backupBatch.docs)}\n`;
```

Comment:
What this `:t` does?

Reply:
`:t` is the log file prefix for "todo" i.e. pending batches